content/W2017/W17-event-ctf.md

```diff
@@ -0,0 +1,26 @@
+title: Security and Networking CTF
+Date: TBD
+Category: Events
+Tags: Security, networking, ctf, events
+Slug: ctf-W17
+Author: Fatima Taj
+Summary: Learn about security and networking!
+
+Are you interested in cyber-security and networking, but don't know where to
+start?
+[Have you ever wondered what exactly a "CTF" is?](https://ctftime.org/ctf-wtf/)
+Do you want to learn how you too can become a Hacker^TM? Come out to WiCS
+Undergrad's Security and Networking CTF (Capture the Flag)! There will be
```

Comment:
add something like "hosted in collaboration with CrySP" and link to the lab https://crysp.uwaterloo.ca/